src/SFA.DAS.AANHub.Application.UnitTests/Notifications/Queries/GetNotificationQueryValidatorTests.cs

```diff
@@ -0,0 +1,132 @@
+﻿using AutoFixture.NUnit3;
+using FluentAssertions;
+using FluentAssertions.Execution;
+using FluentValidation.TestHelper;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.AANHub.Application.CalendarEvents.Queries.GetCalendarEvent;
+using SFA.DAS.AANHub.Application.Employers.Queries;
+using SFA.DAS.AANHub.Application.Notifications.Queries;
+using SFA.DAS.AANHub.Domain.Entities;
+using SFA.DAS.AANHub.Domain.Interfaces.Repositories;
+using SFA.DAS.Testing.AutoFixture;
+using static SFA.DAS.AANHub.Domain.Common.Constants;
+
+namespace SFA.DAS.AANHub.Application.UnitTests.Notifications.Queries;
+public class GetNotificationQueryValidatorTests
+{
+
+    [Test]
+    [RecursiveMoqAutoData]
+    public async Task ValidateCalendarId_Missing_FailsValidation(Member member)
+    {
+        var membersReadRepositoryMock = new Mock<IMembersReadRepository>();
+        membersReadRepositoryMock.Setup(m => m.GetMember(member.Id))
+                                 .ReturnsAsync(member);
+
+        var query = new GetCalendarEventByIdQuery(Guid.Empty, member.Id);
+
+        var sut = new GetCalendarEventByIdQueryValidator(membersReadRepositoryMock.Object);
+
+        var result = await sut.TestValidateAsync(query);
+
+        result.ShouldHaveValidationErrorFor(c => c.CalendarEventId)
+              .WithErrorMessage(GetCalendarEventByIdQueryValidator.CalendarEventIdMissingMessage);
+    }
+
+    [Test, RecursiveMoqInlineAutoData("93231c8f-7645-48c8-90e7-316f3f15c0ed")]
+    public async Task Validate_NotificationIdIsValid(
+        string notificationId,
+        [Frozen] Mock<IMembersReadRepository> membersReadRepository,
+        Member member)
+    {
+        var id = Guid.Parse(notificationId);
+        member.Status = MembershipStatus.Live;
+        membersReadRepository.Setup(m => m.GetMember(member.Id)).ReturnsAsync(member);
+
+        var query = new GetNotificationQuery(id, member.Id);
```

Comment:
Rename id to sound like notificationId as it confuses here with member.Id

Reply:
@DevenKShah is it okay to change this route, I was going off of Alan's documented "/notifications/{id}" route?